### PR TITLE
Adding ability to load individuals from in-memory data as well as off-disk.

### PR DESCRIPTION
**Pull request by julesjacobsen**
Fix for @drseb in issue #57 enabling individuals data to be loaded into the knowledgebase from a ```Map``` rather than off disk.

### CODE REVIEW

**Comment by julesjacobsen** [comment]
One question on naming. Should ```.loadDataFromMap(data)``` really be ```loadIndividuals```? We still have no way of importing labels and presumably this requires different OWLAPI calls.

**Comment by cmungall** [comment]
individualAssociations would be more precise

We need the label loading to get human-friendly results returned. Some 
clients will have their own way of doing a lookup, others will rely on 
the service

```
addAxiom(df.getOWLAnnotationAssertionAxiom(df.getRDFSLabel(), 
ind.getIRI(), df.getOWLLiteral(label))
```

**Comment by julesjacobsen** [comment]
OK so these methods need to change their signatures too. Here 'data' means 'individualAssociations'. Does this make sense for ```loadDataFromOntology``` be changed to  ```loadIndividualAssociationsFromOntology``` given these are actually just being merged? Should there just be a single set of loadOntology or loadInstancedOntology? 

```java
         public Loader loadDataFromOntology(String path) {
            sourceDataBuilder.dataOntology(path);
            return this;
        }

        public Loader loadDataFromOntologies(String... paths) {
            sourceDataBuilder.dataOntologies(paths);
            return this;
        }

        public Loader loadDataFromOntologies(Collection<String> paths) {
            sourceDataBuilder.dataOntologies(paths);
            return this;
        }

        public Loader loadDataFromTsv(String path) {
            sourceDataBuilder.dataTsv(path);
            return this;
        }

        public Loader loadDataFromTsv(String... paths) {
            sourceDataBuilder.dataTsv(paths);
            return this;
        }

        public Loader loadDataFromTsv(Collection<String> paths) {
            sourceDataBuilder.dataTsv(paths);
            return this;
        }

        public Loader loadDataFromMap(Map<String, ? extends Collection<String>> data) {
            sourceDataBuilder.data(data);
            return this;
       }
```

**Comment by julesjacobsen** [comment]
@cmungall / @jnguyenx can you comment on this further or merge this please. We can add the labels in another PR.

File: owlsim-core/src/test/java/org/monarchinitiative/owlsim/io/OntologySourceDataTest.java [hunk 1]
```diff
@@ -18,12 +20,37 @@ public void testItAll() {
         curies.put("MP", "http://purl.obolibrary.org/obo/MP_");
         curies.put("NCBITaxon", "http://purl.obolibrary.org/obo/NCBITaxon_");
 
+        Map<String, List<String>> individuals = new LinkedHashMap<>();
```
Comment (drseb) [inline owlsim-core/src/test/java/org/monarchinitiative/owlsim/io/OntologySourceDataTest.java]: We should now have a thought about how to add additional information to an annotation. E.g. a weight or something. 

Is it already possible to add negation?

**Comment by drseb** [comment]
I think it is good. Thanks. Can't test this right now.

**Comment by julesjacobsen** [comment]
OK, cheers for looking. @drseb I think this should make it possible to do what you need. If not, open another ticket.

**Comment by drseb** [comment]
Thanks. I will test ASAP

**Pull request merged**